script/script.go

```diff
@@ -56,22 +56,30 @@ func (s *Script) Remove(name string) bool {
 	return true
 }
 
-// DisableBuiltinFunction disables a builtin function.
-func (s *Script) DisableBuiltinFunction(name string) {
-	if s.removedBuiltins == nil {
-		s.removedBuiltins = make(map[string]bool)
+// SetBuiltinFunctions allows to define builtin functions.
+func (s *Script) SetBuiltinFunctions(funcs map[string]*objects.BuiltinFunction) {
```

Comment:
I think this should probably be `SetBuiltinFunctions([]*objects.BuiltinFunction)`. I forgot that BuiltinFunction had Name when suggested this signature in the issue 😄.